### PR TITLE
Add support to create tpm ek

### PR DESCRIPTION
**Pull request by leonjia0112**
This function is for creating **ek** key in TPM. This is part of the functions in **init()**.

It is run everytime the node is started.
The resulting error is very sufficient for explaining the running result of creating **ek** key.

What this function does is that it simply run the `createek` command and the rest of the function is for error handling if the execution didn't succeed.

**Test passed. Ready for a review.**
Thanks!

### CODE REVIEW

File: src/tpm.rs [hunk 1]
```diff
@@ -33,6 +33,45 @@ ftpm_initialize.py
...
+fn create_ek() -> Result<(), Box<String>> {
+    let (output, code, _) = run(
+        String::from("createek"),
+        EXIT_SUCCESS,
+        false,
+        false,
+        String::new(),
+    );
+
+    if code.unwrap() != EXIT_SUCCESS {
```
Comment (frozencemetery) [inline src/tpm.rs]: Please prioritize #19.

**Comment by leonjia0112** [comment]
Removed unwrap().

**Test passed. Ready for a review.**
Thanks!

File: src/tpm.rs (hunk 1 above)
Comment (leonjia0112) [reply src/tpm.rs]: removed

File: src/tpm.rs [hunk 2]
```diff
@@ -33,6 +33,54 @@ ftpm_initialize.py
...
+    let (output, code, _) = run(
+        String::from("createek"),
+        EXIT_SUCCESS,
+        false,
+        false,
+        String::new(),
+    );
+
+    if let Some(EXIT_SUCCESS) = code {
+        Ok(())
```
Comment (frozencemetery) [inline src/tpm.rs]: change this to a `return`, then de-indent the `else`.  Repeat below and the code should be much more flat.

**Pull request closed without merging**